lib/cdo/contact_rollups.rb

```diff
@@ -963,4 +949,139 @@ def self.grab_next(s)
   rescue StandardError => error
     log "Error iterating over stream #{s} - #{error}"
   end
+
+  def self.initialize_connections_to_database_clone
```

Comment:
This method (and `create_database_clone`, `delete_database_clone`) seem to be unused code paths. Is there some code missing from this PR?

Reply:
I neglected to commit/push the changes to `build_contact_rollups` that invoke these methods.  That's done now.